manifests/multus-daemonset.do-not-change.yml

```diff
@@ -0,0 +1,410 @@
+---
```

Comment:
The manifest directory should contain manifests used for installation of OVS CNI. This new manifest is only a test helper. Please move it to the cluster/ directory.